documenting.rst

```diff
@@ -1672,11 +1660,6 @@ in production; others are works in progress.
 .. _email_egeakman: mailto:egeakmanegeakman@hotmail.com
 .. _email_garade: mailto:garade@pm.me
 .. _chat_pt_br: https://t.me/pybr_i18n
-.. _doc_ja: https://docs.python.org/ja/
-.. _doc_ko: https://docs.python.org/ko/
-.. _doc_pl: https://docs.python.org/pl/
-.. _doc_zh_cn: https://docs.python.org/zh-cn/
-.. _doc_zh_tw: https://docs.python.org/zh-tw/
```

Comment:
Why removing these?

Reply:
They look too obvious to be mentionned.

Reply:
I found them somewhat handy for two reasons:
1. they show which translations have been already published;
2. they provide an easy way to check the current published translation (instead of having to open d.p.o in a separate tab and selecting the language);

I think they might even deserve their own column.

Reply:
Good idea!